drivers/sensor/maxim/max31865/max31865.c

```diff
@@ -177,15 +177,21 @@ static char *max31865_error_to_string(uint8_t fault_register)
 static int max31865_fault_register(const struct device *dev)
 {
 	uint8_t fault_register;
+	uint8_t old_d5_d3_d2;
 
 	max31865_spi_read(dev, (REG_FAULT_STATUS), &fault_register, 1);
 	struct max31865_data *data = dev->data;
+	old_d5_d3_d2 = data->config_control_bits & 0b00101100;
```

Comment:
I believe the use of binary constants is discouraged. See #60564.
I suggest to change it to Hex and, perhaps, hide it behind a macro-define.

Reply:
Done !